contingency/contingency-api/src/main/java/com/powsybl/contingency/ContingencyContext.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.powsybl.contingency;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Objects;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ * @author Etienne Lesot <etienne.lesot at rte-france.com>
+ * <p>
+ * provide the context to get information of the network after a security analysis
```

Comment:
We should have a consistent vocabulary in this API. Evrywhere we use pre/post contingency so we should use sames in doc.

Reply:
I changed the javadoc to use pre/post contingency. Did I miss something or was it not what you were referring to?

Reply:
Ok sorry I was reading old version of the javadoc.